src/MLTable/MLTable.js

```diff
@@ -304,10 +304,21 @@ class MLTable extends React.Component {
         return null
       }
       const showIcon = record.children || hasNestedTables
+
+      let testId = 'mltable-expand'
+      if (record.hasOwnProperty('name')) {
+        let parseText = record['name'].split(',')
+        let entityName = parseText[0]
+        testId = `mltable-expand-${entityName}`
+      } else if (record.hasOwnProperty('propertyName')) {
+        testId = `mltable-expand-${record['propertyName']}`
+      }
+
```

Comment:
Looks good. So any row in MLTable with expand icon will get a test id with property name appended to it.

Reply:
Yeah, I've updated the "name" selector to "entityName" because I thought "name" was too generic for a custom id selector. I will refactor the Hub Central code to support this.